docs/developer-guide.md

```diff
@@ -34,3 +38,22 @@ spark-submit \
   <path-to-microsoft-spark-jar> \
   <path-to-your-app-exe> <argument(s)-to-your-app>
 ```
+
+## How to Support New Spark Releases
+
+### New patch version release
+Apache Spark's new patch release involves only the internal changes such as bug fixes, etc. Thus, it is straightfoward to support a new patch version release.
```

Comment:
```suggestion
Since Apache Spark's [MAINTENANCE] releases involve only internal changes (e.g., bug fixes etc.), it is straightforward to upgrade the code base to support a [MAINTENANCE] release. The steps to do this are below:
```